### ISSUE TITLE
Can't to use "id" as a string in the case that it looks like a natural number

### ISSUE THREAD

**Issue by feonit** [typicode/json-server#374]
I have:

```
"regions": [
    {
      "id": "177",
      "name": "Moscow"
    }
]
```

but the request:

`GET /regions/177`

is empty

If i remove code from test/server/units.js:

```
    } else if (!isNaN(+value)) {
      return +value
    }
```

to:

```
    } else {
      return value
    }
```

it works fine. What is wrong?

**Comment by typicode** [comment]
Hi @feonit,

You have to use an integer for the `id`. For example, ` { "id": 177 }` should work fine.
I'm working on improving that on the `next` branch, so that `"177"` can be used as well.

But it's not yet ready to be published. I'll update this issue when it'll be available on `npm`.

**Comment by typicode** [comment]
I've published `v0.9.0-beta.2`. It should let you use `id` as string.
It's a beta so there may be some other issues, let me know if you find anything else.

To install it add `@next`

```
npm install -g json-server@next
```

`README` can be found in `next` branch.

**Comment by typicode** [comment]
Published `v0.9.0` with the fix
